streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -542,6 +552,39 @@ public void shouldPropagateExceptionFromStandbyCommit() {
         verify(standby);
     }
 
+    @Test
+    public void shouldSendPurgeData() {
```

Comment:
should we add a test that also checks that a purge request is sent after a previous one has completed?